packages/protocol/contracts/common/UsingRegistry.sol

```diff
@@ -10,6 +10,7 @@ import "./interfaces/IFeeCurrencyWhitelist.sol";
 import "./interfaces/IFreezer.sol";
 import "./interfaces/IRegistry.sol";
 import "./interfaces/ICeloUnreleasedTreasure.sol";
+import "./interfaces/IEpochManager.sol";
```

Comment:
Is this intentional?

Ditto for V2

Reply:
probaly added during a branch merge. fixed